### ISSUE TITLE
@trezor/connect-common can't resolve 0.0.32

### ISSUE THREAD

**Issue by own2pwn** [trezor/trezor-suite#12326]
@trezor/connect-common@npm:0.0.32: No candidates found

https://github.com/trezor/trezor-suite/blob/bc18e7ad8221fdcf518028876f2e3c20a0db2877/packages/connect-common/package.json#L3

**Comment by karliatto** [comment]
@own2pwn thank you for your comment, this is result of some testing on our release process. 
I am curious, how did you come to use that package? Are you using the beta version of `@trezor/connect` ?

**Comment by own2pwn** [comment]
hey @karliatto , it came from `@solana/wallet-adapter-wallets`. not sure though how it resolved in 0.0.32. 

```
❯ yarn add @solana/wallet-adapter-wallets
➤ YN0000: · Yarn 4.0.1
➤ YN0000: ┌ Resolution step
➤ YN0082: │ @trezor/connect-common@npm:0.0.32: No candidates found
➤ YN0000: └ Completed in 4s 14ms
➤ YN0000: · Failed with errors in 4s 32ms
```

https://github.com/anza-xyz/wallet-adapter/blob/a8e4bdee26b4e4d1dd6b321999eb57c8516d775a/packages/wallets/trezor/package.json#L39

**Comment by karliatto** [comment]
That is not normal behavior since  the version of `@trezor/connect-web` [9.2.3](https://www.npmjs.com/package/@trezor/connect-web/v/9.2.3) is tagged as beta in NPM and that should not be updated automatically.

Besides if you looks at the `pnpm-lock.yaml` file in the same project you mentioned it uses `@trezor/connect-common` 0.0.30  https://github.com/anza-xyz/wallet-adapter/blob/a8e4bdee26b4e4d1dd6b321999eb57c8516d775a/pnpm-lock.yaml#L7593

And if I just try to run `yarn add @trezor/connect-web` it installs `"@trezor/connect-common@0.0.31"` which is right because `0.0.31` is tagged as lates in NPM and is the current stable release.
